website/content/docs/secrets/pki/cmpv2.mdx

```diff
@@ -0,0 +1,130 @@
+Batch tokens are required to avoid an excessive amount of leases being generated
+and persisted as every CMPv2 incoming request needs to be authenticated.
+
+The path within an ACL policy must match the `cmp` path underneath the 
+PKI mount.  The path to use can be the default `cmp` path or a role based one.
+
+If using the `sign-verbatim` as a path policy, the following
+ACL policy will allow an authenticated client access the required PKI CMP path.
+```
+path “pki/cmp” {
+  capabilities=[“update”, “create”]
+}
+```
+
+For a role base path policy, this sample policy can be used
+```
+path “pki/roles/my-role-name/cmp” {
+  capabilities=[“update”, “create”]
+}
+```
+
+#### Updating the PKI mount tunable parameters
+
+Once the authentication mount has been created and configured, the authentication mount's accessor
+will need to be captured and added within the PKI mount's [delegated auth accessors](/vault/api-docs/system/mounts#delegated_auth_accessors).
+
+To get an authentication mount's accessor field, the following command can be used.
+
+```shell-session
+$ vault read -field=accessor sys/auth/auth/cert
+```
+
+For CMP to work within certain clients, a few response headers need to be explicitly allowed
+along with configuring the list of accessors the mount can delegate authentication towards.
+The following will grant the required response headers, you will need to replace the values for the `delegated-auth-accessors`
+to match your values.
+
+```shell-session
+$ vault secrets tune \
+  -allowed-response-headers="Content-Transfer-Encoding" \
+  -allowed-response-headers="Content-Length" \
+  -allowed-response-headers="WWW-Authenticate" \
+  -delegated-auth-accessors="auth_cert_4088ac2d" \
+  pki
+```
+
+#### Enabling CMPv2
+
+Enabling CMP is a matter of writing to the `config/cmp` endpoint, to set it
+enabled and configure default issuance policy and authentication.
+
+```shell-session
+vault write pki/config/cmp -<<EOC
+{
+  "enabled": true,
+  "authenticators": {
+    "cert": {
+      "accessor": "auth_cert_4088ac2d"
+    }
+  }
+}
+EOC
+```
+
+Of course, substituting your own role and accessor values.  After this, the
+CMP endpoints will be available to handle client requests, authenticated with the 
+previously configured Cert Auth method.
+
+## Limitations
+
+The initial release of CMPv2 support is intentionally limited to a subset of the
+protocol, covering Initialization, Certification, and Key Update, over HTTP.
+In particular, the following are not yet supported:
+
+ * Basic authentication scheme using PasswordBasedMac
+ * Revocation
+ * CRL fetching via CMP itself.
+ * CA creation/update operations.
+
+In addition, CMPv2 will not yet be integrated with these existing Vault PKI features:
+ 
+ * Certificate Metadata - CMPv2 has no means of providing metadata.
+ * Certificate Issuance External Policy Service [(CIEPS)](/vault/docs/concepts/pki/cieps) - 
+   May be implemented in the future.
```

Comment:
I suggest either making this a table and providing the reasoning for both items, or leaving it a list and just noting the affected features without noting a reason. Saying something "may be implemented in the future" isn't helpful to the reader.